runtime/vm/visible.c

```diff
@@ -231,6 +318,8 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 		/* If a problem occurred in nest host verification then the nest host value is invalid */
 		if  ((J9_VISIBILITY_ALLOWED == result) && (NULL != nestHost)) {
 			clazz->nestHost = nestHost;
+		} else if (throwException) {
```

Comment:
if `result == J9_VISIBILITY_ALLOWED` but `nestHost == NULL`, won't the `setNestmatesIncompatibleClassChangeError()` assert?

In what case would visibility be allowed by the nestHost end up null?

Reply:
I'm not sure I see a path where this can happen which means the `if  ((J9_VISIBILITY_ALLOWED == result) && (NULL != nestHost)) ` is incorrect